examples/omni-lock-unisat/README.md

```diff
@@ -0,0 +1,21 @@
+# Lumos Works with UniSat
+
+Nervos maintains a powerful lock
+called [Omni Lock](https://github.com/XuJiandong/docs-bank/blob/master/omni_lock.md) (previously named RC lock), which
```

Comment:
Why not link it to the RFC at https://github.com/nervosnetwork/rfcs/blob/master/rfcs/0042-omnilock/0042-omnilock.md